packages/react-devtools-extensions/src/background.js

```diff
@@ -2,17 +2,41 @@
 
 'use strict';
 
+import {IS_FIREFOX} from './utils';
+
 const ports = {};
 
-const IS_FIREFOX = navigator.userAgent.indexOf('Firefox') >= 0;
+if (!IS_FIREFOX) {
+  // Manifest V3 method of injecting content scripts (not yet supported in Firefox)
```

Comment:
Hi. Not sure if this is the best place to mention it but this is now supported in Firefox and, in fact, feature detection works generally better than user-agent detection. This extension should be trivial to "port" on Firefox MV3 ;-)

Reply:
Thanks for letting me know! Unfortunately we are extremely understaffed now, so we won't be able to improve it soon. I'll make it as a TODO and hope we'll find time.